src/style.css

```diff
@@ -0,0 +1,38 @@
+html {
+  --star_acive: url(images/star-active.svg);
+  --star: url(images/star.svg);
+}
+
+ul,
+li,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  list-style: none;
+  background-position: center;
+  object-fit: cover;
+  display: flex;
+  background-repeat: no-repeat ;
+  width: 16px;
+  height: 16px;
+  background-image: var(--star);
```

Comment:
read about positioning of your styles inside block, it's preferably to set display, width, margin etc. on the top of block, and then other styles